Dungeoneer/assets/shaders/animate.frag

```diff
@@ -16,17 +16,15 @@ varying vec2 v_texCoords;
 varying float v_fogFactor;
 varying float v_eyeDistance;
 
-vec4 tex_Color;
-
 void main() {
-  vec4 color;
+    float epsilon = 1.0 / 256.0;
```

Comment:
This might be better off as a constant, like:
`const float c_epsilon = 1.0 / 256.0;`

Reply:
Sounds reasonable to me. I've made the change.